test/src/main/java/org/apache/accumulo/test/functional/ConcurrentDeleteTableIT.java

```diff
@@ -52,7 +52,7 @@
 
 public class ConcurrentDeleteTableIT extends AccumuloClusterHarness {
 
-  @Test
+  @Test(timeout = 3 * 60 * 1000)
```

Comment:
In the super class, we have a method that can be overridden called `defaultTimeoutSeconds`, which probably would work better for this. The timeout for that is scalable on the command-line with `-Dtimeout.factor=2` or similar. See `SplitIT.java` for an example.